<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cameraCalibration</class>
 <widget class="QWidget" name="cameraCalibration">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Avenir Next</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Reflectance Transform Imaging</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="Description" stdset="0">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="vBoxLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="titleHLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="titleLbl">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>450</width>
           <height>20</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Avenir Next</family>
           <pointsize>27</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>Camera Calibration</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QGridLayout" name="cornersGridLayout">
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="descHLayout">
         <item>
          <widget class="QLabel" name="descCornersLbl">
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;Select the number of corners inside the &lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;calibration object&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="inputCornersVLayout">
         <item>
          <layout class="QHBoxLayout" name="xAxisHLayout">
           <item>
            <widget class="QLabel" name="xAxisLbl">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;X-Axis coners&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="xAxisInput"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="yAxisHLayout">
           <item>
            <widget class="QLabel" name="yAxisLbl">
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Y-Axis corners&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="yAxisInput"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="uploadGridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="stCamHLayout">
         <item>
          <widget class="QLabel" name="stCamLbl">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;Upload static camera calibration video&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;(camera with no light on)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stCamBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Upload video</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="mvCamHLayout">
         <item>
          <widget class="QLabel" name="mvCamLbl">
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;Upload moving camera calibration video&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;(camera with direct light on the object) &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mvCamBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Upload video</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="startBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Avenir Next</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="accessibleDescription">
        <string/>
       </property>
       <property name="text">
        <string>Start Calibration</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
